components/automate-chef-io/README.md

```diff
@@ -81,11 +88,314 @@ This will create the file and fill in the necessary frontmatter automatically.
 
 Hugo allows us to nest our data directory structure as much as necessary. You can add as many folders as necessary under `components/automate-chef-io/data/docs`.
 
-```
+```output
 .
 ├── data
 │   ├── docs
 │   │   ├── cli
 |   │   │   ├── command_one.yml
 |   │   │   └── command_two.yml
 ```
+
+## API Documentation
+
+To view the API documentation locally, 1. From the `components/automate-chef-io` directory, run the following command to start the development server.
+
+```shell
+$ make serve
+```
+
+To view the API docs in your browser, navigate to:
+
+```shell
+http://localhost:1313/docs/api
+```
+
+### Building API Docs
+
+Chef Automate  generates API documentation from protoc files comments using
+
+* [grpc-gateway protoc-gen-swagger](https://github.com/grpc-ecosystem/grpc-gateway/tree/master/protoc-gen-swagger) as the protoc compiler
+* [OpenAPI 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) as the API schema
+* [ReDoc](https://github.com/Redocly/redoc) as the front-end display
+
+To build the API documentation during development:
+
+1. Make any edits to the .proto files necessary.
+2. Recompile the .proto files as described in the [Automate development guide](https://github.com/chef/automate/blob/master/dev-docs/DEV_ENVIRONMENT.md)
+3. Sync the .swagger files generated with those in the docs component: `make sync_swagger_files`
```

Comment:
This is somewhat ambiguous, it should mention being in the `components/automate-chef-io` directory while running this command.